constants/Bingo.json

```diff
@@ -1,320 +1,324 @@
+                "in the End and hold it while wearing",
+                "some Ender Armor and/or Equipment."
+            ]
+        },
+
+        "A Best Friend": {
+            "difficulty": "Medium",
+            "note": [
+                "Craft any pet"
+            ],
+            "guide": [
+                "Rabbit, Jerry or Pig pet",
+                "will be your best options.",
+                "You will need an Enchanted Egg.",
+                "(requires 2.5k Raw Chicken Collection",
+                "and 144 Eggs to craft)"
+            ]
+        },
+
+        "Farmer": {
+            "difficulty": "Medium",
+            "note": [
+                "Reach Farming Level 20"
+            ],
+            "guide": [
+                "You will most likely get this while",
+                "going for the Carrot Collector goal."
+            ]
+        },
+
+        "Healthy": {
+            "difficulty": "Easy",
+            "note": [
+                "500 Health"
+            ],
+            "guide": [
+                "Wear Mushroom armor enchanted with Growth 5 ",
+                "(Enchanting Level 5) during the Night.",
+                "Applying Growth 5 costs 50 Levels each."
+            ]
+        },
+
+        "Rotten Flesh Collector": {
+            "difficulty": "Medium",
+            "note": [
+                "47.5k Rotten Flesh"
+            ],
+            "guide": [
+                "Unlock the Zombie Minion by killing",
+                "Zombies in the Graveyard in The Hub.",
+                "Place as many minions as you can",
+                "give them a medium storage and the best fuel",
+                "you can. Collect them in time."
+            ]
+        },
+
+        "Fairy Finder": {
+            "difficulty": "Easy",
+            "note": [
+                "25 Fairy Souls"
+            ],
+            "guide": [
+                "Collect those you run along while doing other goals.",
+                "If you have trouble check TimeDeo's fairy soul",
+                "tutorial on YouTube or run /neusouls on to get",
+                "the locations shown."
+
+            ]
+        },
+
+        "Friend Of The Seas": {
+            "difficulty": "Hard",
+            "note": [
+                "Equip an Uncommon or higher Dolphin"
+            ],
+            "guide": [
+                "You have to fish up 250 Sea Creatures",
+                "to get a Common Dolphin. Upgrade it",
+                "with Kat for 100k coins and wait a day.",
+                "You might want to unlock Angler Armor for this."
+            ]
+        },
+
+        "I See Stars": {
+            "difficulty": "Easy",
+            "note": [
+                "5 star a dungeon item"
+            ],
+            "guide": [
+                "Get yourself the Rogue Sword from Jamie",
+                "in The Hub and star it using the Gold Essence",
+                "from the Gold Forger Visitor in your Garden."
```

Comment:
Hmm. The essence cost for this is fairly low, but it still takes between 3 and 8 Gold Forger visits. Another option would be killing Endermen and salvaging Ender Armor for Dragon Essence and 5 starring a Void Sword. It does require more essence, but the drops are fairly common if you get a godsplash.